device-types/Citrix/MPX-8200.yaml

```diff
@@ -0,0 +1,48 @@
+---
+manufacturer: Citrix
+model: MPX-8200
+slug: citrix-mpx-8200
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Console
+    type: de-9
+power-ports:
+  - name: PS 1
+    type: iec-60320-c14
+    maximum_draw: 450
+  - name: PS 2
+    type: iec-60320-c14
+    maximum_draw: 450
```

Comment:
It appears that these are modular PSU's. Please move the definition to `module-bays` and create the associated PSUs within the module-types folder